plugins/@grouparoo/customerio/__tests__/export/destination-mapping-options.ts

```diff
@@ -0,0 +1,82 @@
+import path from "path";
+import "@grouparoo/spec-helper";
+import { helper } from "@grouparoo/spec-helper";
+import {
+  destinationMappingOptions,
+  getUserFields,
+} from "../../src/lib/export/destinationMappingOptions";
+import { loadAppOptions, updater } from "../utils/nockHelper";
+import { connect } from "../../src/lib/connect";
+
+const nockFile = path.join(
+  __dirname,
+  "../",
+  "fixtures",
+  "destination-mapping-options.js"
+);
+
+// these comments to use nock
+const newNock = false;
+require("./../fixtures/destination-mapping-options");
+// or these to make it true
+// const newNock = true;
+// helper.recordNock(nockFile, updater);
+
+const appOptions = loadAppOptions(newNock);
+const appId = "app_78189023490-dfsjklfdsklj90-90-3k";
+
+async function runDestinationMappingOptions({}) {
+  return destinationMappingOptions({
+    appOptions,
+    app: null,
+    appId: appId,
+    connection: null,
+    destination: null,
+    destinationId: null,
+    destinationOptions: null,
+  });
+}
+
+describe("sendgrid/destinationMappingOptions", () => {
```

Comment:
still from sendgrid. 
This should be an easy test to do seeing has how there are no API requests.